app/src/main/java/org/mozilla/fenix/onboarding/DefaultBrowserNotificationWorker.kt

```diff
@@ -90,7 +90,7 @@ class DefaultBrowserNotificationWorker(
             val channel = NotificationChannel(
                 NOTIFICATION_CHANNEL_ID,
                 applicationContext.getString(R.string.notification_marketing_channel_name),
-                NotificationManager.IMPORTANCE_DEFAULT
+                NotificationManager.IMPORTANCE_LOW
             )
```

Comment:
Random drive-by 🚗: Afaik a `Channel` needs to be recreated for the importance changes to take effect. Otherwise this affects only new users.

Reply:
Isn't this channel used just for this one notification? There might not actually be many existing users; only the ones upgrading in the first 24 hours after initial setup.

How does one go about recreating a `Channel`?